bin/ci-builder

```diff
@@ -16,7 +16,7 @@
 
 set -euo pipefail
 
-NIGHTLY_RUST_DATE=2024-01-01
+NIGHTLY_RUST_DATE=2024-03-25
```

Comment:
This is causing the nightly bootstrapping to fail: https://buildkite.com/materialize/tests/builds/79155#018e7b86-de2a-4251-aed5-1bcb442f22dd
You probably have to upgrade some libraries to not use stdsimd